offscreen/offscreen.html

```diff
@@ -7,5 +7,6 @@
   <body>
     <script type="text/javascript" src="./load-offscreen.js"></script>
     <iframe src="./trezor-iframe.html"></iframe>
+    <iframe src="./ledger-iframe.html"></iframe>
```

Comment:
Would this security hardening work?
```suggestion
    <iframe sandbox="" src="./trezor-iframe.html"></iframe>
    <iframe sandbox="" src="./ledger-iframe.html"></iframe>
```
Or does it clash with the rest of the logic introduced in this PR? Could be a nice touch. Weird, because the origin of the content of the iframe is at a similar trust level of the code that loads it, but if such isolation costs us nothing, it could only add security without giving up on anything.

Reply:
adding this property caused two things: Before the iframe would show up in the extension manifest in the extension manager, and it received communication from the offscreen document. After setting it both those things stopped working. For those reasons I did not add this hardening to the PR

Reply:
Fair enough, that was a long shot - thanks for trying!
I'm not too worried about that honestly, would have been nice but far from required